src/main/java/com/salesforce/ConsumeTopic.java

```diff
@@ -78,7 +78,9 @@ public Exception call() {
             KafkaConsumer<Integer, byte[]> consumer = new KafkaConsumer<>(consumerConfigForTopic);
             TopicPartition topicPartition = new TopicPartition(topicName, 0);
             consumer.assign(Collections.singleton(topicPartition));
-            consumer.seekToBeginning(Collections.singleton(topicPartition));
+            // Always start from end, since otherwise we'll play the catch up game.
+            // Our goal here is not to monitor the lag.
+            consumer.seekToEnd(Collections.singleton(topicPartition));
```

Comment:
If this is false we shouldn't do it: https://github.com/lyft/kafka-partition-availability-benchmark/blob/master/src/main/resources/kafka-partition-availability-benchmark.properties#L20

This is a good default if producers are active :+1: